--- cmake/Modules/FindvecLib.cmake	2016-11-01 21:21:52.000000000 -0600
+++ cmake/Modules/FindvecLib.cmake	2016-11-01 21:42:30.000000000 -0600
@@ -12,10 +12,10 @@
 
 set(__veclib_include_suffix "Frameworks/vecLib.framework/Versions/Current/Headers")
 
-find_path(vecLib_INCLUDE_DIR vecLib.h
+find_path(vecLib_INCLUDE_DIR cblas.h
           DOC "vecLib include directory"
-          PATHS /System/Library/${__veclib_include_suffix}
-                /System/Library/Frameworks/Accelerate.framework/Versions/Current/${__veclib_include_suffix}
+          PATHS /System/Library/Frameworks/Accelerate.framework/Versions/Current/${__veclib_include_suffix}
+                /System/Library/${__veclib_include_suffix}
                 /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk/System/Library/Frameworks/Accelerate.framework/Versions/Current/Frameworks/vecLib.framework/Headers/)
 
 include(FindPackageHandleStandardArgs)
