--- numpydoc/docscrape.py
+++ numpydoc/docscrape-new.py
@@ -114,6 +114,7 @@
             'Other Parameters': [],
             'Attributes': [],
             'Methods': [],
+            'Side Effects': [],
             'See Also': [],
             'Notes': [],
             'Warnings': [],
@@ -432,9 +433,10 @@
         for param_list in ('Parameters', 'Returns', 'Yields',
                            'Other Parameters', 'Raises', 'Warns'):
             out += self._str_param_list(param_list)
+        out += self.__str_side_effects()
         out += self._str_section('Warnings')
         out += self._str_see_also(func_role)
-        for s in ('Notes', 'References', 'Examples'):
+        for s in ('Side Effects', 'Notes', 'References', 'Examples'):
             out += self._str_section(s)
         for param_list in ('Attributes', 'Methods'):
             out += self._str_param_list(param_list)
